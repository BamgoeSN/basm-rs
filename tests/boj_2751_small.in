5
1000000
-1000000
999999
0
-461