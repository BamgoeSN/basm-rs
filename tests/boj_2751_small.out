-1000000
-461
0
999999
1000000